src/executables/uber.js

```diff
@@ -8,4 +8,10 @@ program.version(pkg.version)
   .description('Figure out if you should order a car to pick you up and drive you to where you want to go')
   .command('price', 'get price estimate')
   .command('time', 'get time to pickup estimate')
+  .arguments('<command>')
+    .action((cmd) => {
+      program.outputHelp();
+      console.log(`\n Unknown command ${cmd}.`));
```

Comment:
It looks like there's technically a leading space before `Unknown`? If so, is there a reason for this leading space?

Reply:
Just personal preference.